tfjs-backend-webgpu/src/from_pixels_webgpu_test.ts

```diff
@@ -56,6 +56,11 @@ describeWebGPU('fromPixels', () => {
        if ('requestVideoFrameCallback' in video) {
          // tslint:disable-next-line:no-any
          await new Promise(go => (video as any).requestVideoFrameCallback(go));
+       } else {
+         // On mobile safari the ready state is ready immediately.
+         if (video.readyState < HTMLVideoElement.prototype.HAVE_CURRENT_DATA) {

```

Comment:
WebGPU uses video.currentTime for check: https://github.com/gpuweb/cts/blob/0dd7405d1bfa950a71572aeb77aa38aacef6653d/src/webgpu/web_platform/util.ts#L48. 
WebGL uses a simpler solution as currently we have: https://github.com/KhronosGroup/WebGL/blob/main/sdk/tests/js/webgl-test-utils.js#L3182.

Reply:
Digging through the history of WebGL CTS, it seems awaiting `video.play()` + optional rVFC works fine across browsers and is preferred over `video.currentTime`, so I decided to drop the change here.